node/forest_libp2p/src/behaviour.rs

```diff
@@ -156,11 +145,12 @@ impl<TSubstream: AsyncRead + AsyncWrite + Unpin + Send + 'static>
     }
 }
 
-impl<TSubstream: AsyncRead + AsyncWrite + Send + Unpin + 'static> ForestBehaviour<TSubstream> {
+impl ForestBehaviour {
     /// Consumes the events list when polled.
     fn poll<TBehaviourIn>(
         &mut self,
         _: &mut Context,
+        _: &mut impl libp2p::swarm::PollParameters,
```

Comment:
I just pasted this in without removing "libp2p::swarm" (my bad). Just import PollParameters at the top.